time/errata.html

```diff
@@ -41,8 +41,14 @@ <h2 id="known-errors">Known errata in Time Ontology in OWL</h2>
           <dd>Refers to: <a href="https://www.w3.org/TR/2017/REC-owl-time-20171019/#topology">3.1 Topological Temporal Relations</a></dd>
           <dd>Description: As explained in the text, the domain of the <code>hasTemporalDuration</code> property is <code>TemporalEntity</code>. However, the <code>hasTemporalDuration</code> property does not appear in the model diagram for the <code>TemporalEntity</code> class.</dd>
           <dd>Correction: Add <code>hasTemporalDuration : TemporalDuration</code> to the model of the <code>TemporalEntity</code> class in Figure 1.</dd>
+          <dt><a id="entry-2">2. Alignment with OGC terminology for Temporal Coordinate Reference System</dt>
+          <dd>Added: 28 Feb 2019</dd>
+          <dd>Type: Editorial</dd>
+          <dd>Refers to: <a href="https://www.w3.org/TR/owl-time/#trs-clock-calendar">3.2 Temporal reference systems, clocks, calendarss</a> and <a href="https://www.w3.org/TR/owl-time/#time:TRS">4.1.18 Temporal reference system</a></dd>
+          <dd>Description: The terminology "Temporal Coordinate System" taken from from ISO 19108 has been superseded by "Temporal Coordinate Reference System" in ISO 19111:2019 (also published as OGC Abstract Specification - Topic 2)</dd>
```

Comment:
s/from from/from

Reply:
Oh dear. No I have no idea ... is it some kind of whitespace normalization?